### PR TITLE
Add configureInstrumentations for users to customize instrumentations

### PR DESCRIPTION
**Pull request by lpsinger**

### CODE REVIEW

**Review by codeboten — COMMENTED** [review]
Thanks for the contribution! Please sign the CLA

**Comment by lpsinger** [comment]
> Thanks for the contribution! Please sign the CLA

It's with my organization right now.

**Comment by lpsinger** [comment]
It's signed now!

File: nodejs/packages/layer/src/wrapper.ts [hunk 1]
```diff
@@ -69,6 +70,7 @@ const instrumentations = [
   new NetInstrumentation(),
   new PgInstrumentation(),
   new RedisInstrumentation(),
```
Comment (Aneurysm9) [inline nodejs/packages/layer/src/wrapper.ts]: Should these all be moved into a default implementation of `configureInstrumentations()` such that users have a way to ask for less auto-instrumentation and not just more?  Minimizing the impact of auto-instrumentation on cold start times is a recurring theme we're hearing from customers.

Comment (lpsinger) [reply nodejs/packages/layer/src/wrapper.ts]: Suits me. Done.

Note, I have no good way to test this.

**Review by Aneurysm9 — APPROVED** [review]

**Comment by lpsinger** [comment]
Thank you! When will this change take effect in the Lambda layers on AWS that are listed at https://aws-otel.github.io/docs/getting-started/lambda/lambda-js?

**Comment by lpsinger** [comment]
Hi, when are you going to upload a new Lambda layer?

**Pull request merged**